cmd/nodecmd/create_aws.go

```diff
@@ -341,3 +349,21 @@ func addCertToSSH(certName string) error {
 	utils.SetupRealtimeCLIOutput(cmd, true, true)
 	return cmd.Run()
 }
+
+// checkRegions checks if the given regions are available in AWS.
+func checkRegions(regions []string) (bool, error) {
+	regionChecker, err := getAWSCloudCredentials(awsProfile, "us-east-1")
```

Comment:
the return of getAWSCloudCredentials is not a regionChecker object

Reply:
us-east-1 can be made a constant

Reply:
it's a object that we only use to make AWS API call to check region. I am open to a better name. 
moved us-east-1 to the const

Reply:
call it awsCloud probably, it is more directly related to the data type

Reply:
fixed name